handler.go

```diff
@@ -19,13 +19,13 @@ func newBaseHandler(name string, broadcast BroadcastAdaptor) *baseHandler {
 	}
 }
 
-// On registers the function f to handle message.
-func (h *baseHandler) On(message string, f interface{}) error {
+// On registers the function f to handle an event.
+func (h *baseHandler) On(event string, f interface{}) error {
```

Comment:
Looking at the examples here: http://socket.io/docs/, they call the first argument an event, but the entire package (event and payload) a message.